code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -292,9 +292,11 @@
 
 /obj/item/weapon/reagent_containers/pill/zoom/Initialize()
 	. = ..()
-	reagents.add_reagent("impedrezene", 10)
-	reagents.add_reagent("synaptizine", 5)
-	reagents.add_reagent("hyperzine", 5)
+	if(prob(50))						//VOREStation edit begin: Zoom pill adjustments
+		reagents.add_reagent("mold", 2)	//Chance to be more dangerous
+	reagents.add_reagent("expired_medicine", 5)
+	reagents.add_reagent("fertilizer", 5)

```

Comment:
Not sure I agree with fertilizer being here, though. Makes no sense.

Reply:
Admittedly, I just was looking for a weaker addition to presense of expired medicine, to make toxin buildup not as immedeate and bit more progressive, and only 'weak and slow processing' toxin I could find was fertilizer (which i believe is the weakest toxin in the game). I could make a renamed version of it, just tried to use what already was in the code.

Reply:
I'd rather not use fertilizer, and having one new reagent just for this purpose is kind of dumb.

I suggest just deleting the line outright. We have enough sensible and detrimental reagents here already to balance it.

Reply:
Fixed